src/reactapp/src/utils/price.js

```diff
@@ -1,32 +1,42 @@
+import env from './env';
 import RootElement from './rootElement';
 
+const formatMapper = currencySymbol => ({ type, value }) => {
+  switch (type) {
+    case 'currency':
+      return currencySymbol || value;
+    case 'minusSign':
+      return '- ';
+    case 'plusSign':
+      return '+ ';
+    default:
+      return value;
+  }
+};
+
 export const formatPrice = (value, showSign) => {
   const {
-    code: currencyCode,
-    symbol: currencySymbol,
+    code: rootCurrencyCode,
+    symbol: rootCurrencySymbol,
   } = RootElement.getCurrency();
+  const envCurrencyCode = env.currencyCode;
+  const envCurrencySymbol = env.currencySymbol;
 
-  var formatter = new Intl.NumberFormat(document.documentElement.lang, {
+  const currencyCode = envCurrencyCode || rootCurrencyCode;
+  const currencySymbol = envCurrencySymbol || rootCurrencySymbol;
+
+  const formatter = new Intl.NumberFormat(document.documentElement.lang, {
     style: 'currency',
     currency: currencyCode,
     signDisplay: showSign ? 'always' : 'auto',
   });
 
-  return typeof Intl.NumberFormat.prototype.formatToParts === 'function'
-    ? formatter
-        .formatToParts(value)
-        .map(({ type, value }) => {
-          switch (type) {
-            case 'currency':
-              return currencySymbol || value;
-            case 'minusSign':
-              return '- ';
-            case 'plusSign':
-              return '+ ';
-            default:
-              return value;
-          }
-        })
-        .reduce((string, part) => string + part)
-    : formatter.format(value);
+  if (typeof Intl.NumberFormat.prototype.formatToParts === 'function') {
+    formatter
```

Comment:
You need to return this value, otherwise the default formatter below is always returned instead of this formatting. The original method had a conditional return, that's now gone.

Reply:
ah.. i missed it. thanks to dig that out :)